app/views/step_by_step_pages/index.html.erb

```diff
@@ -28,56 +28,83 @@
 <div class="govuk-grid-row govuk-!-margin-top-5">
   <div class="govuk-grid-column-one-quarter">
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full">
-        <%= render "govuk_publishing_components/components/input", {
-          label: {
-            text: "Title or base path"
-          },
-          name: "search",
-          id: "filterTableInput"
-        } %>
-      </div>
-      <div class="govuk-grid-column-full">
-        <%= render "govuk_publishing_components/components/select", {
-          id: "filterStatus",
-          label: "Status filter",
-          full_width: true,
-          options: [
-            {
-              text: "All",
-              data_attributes: {
-                show: "all"
-              },
-              selected: true
-            },
-            {
-              text: "Draft",
-              data_attributes: {
-                show: "draft"
-              }
-            },
-            {
-              text: "Live",
-              data_attributes: {
-                show: "live"
-              }
-            },
-            {
-              text: "Scheduled",
-              data_attributes: {
-                show: "scheduled"
-              }
+      <%= form_tag({}, enforce_utf8: false, method: :get) do %>
+        <div class="govuk-grid-column-full">
+          <%= render "govuk_publishing_components/components/input", {
+            label: {
+              text: "Title or URL",
+              bold: true
             },
-            {
-              text: "Unpublished changes",
-              data_attributes: {
-                show: "unpublished_changes"
+            name: "title_or_url",
+            value: params[:title_or_url],
+            id: "title_or_url"
+          } %>
+        </div>
+        <div class="govuk-grid-column-full">
+          <%= render "govuk_publishing_components/components/label", {
+            text: "Status",
+            html_for: "status",
+            bold: true
+          } %>
+          <%= render "govuk_publishing_components/components/select", {
+            id: "status",
+            name: "status",
+            label: "",
+            full_width: true,
+            value: params[:status],
+            options: [
+              {
+                text: "All",
+                data_attributes: {
+                  show: "all"
+                },
+              },
+              {
+                text: "Draft",
+                value: "draft",
+                data_attributes: {
```

Comment:
I don't think these `data_attributes` are needed anymore - only the `value` gets used in generating the filter URL 🤔

Reply:
I thought `data_attributes` were for tracking? https://github.com/alphagov/govuk_publishing_components/blob/3fdfd33366502f61d548f45399b026860e3ffafd/lib/govuk_publishing_components/presenters/select.rb#L11

Reply:
Ah, true 👍